WordPress/gutenberg
# [RNMobile] Update icon for UngroupButton component

Part of #50520
Related to #17575

## What?
This PR updates the ungroup icon that appears on the toolbar at the bottom of the screen.

![ungroup](https://github.com/WordPress/gutenberg/assets/54422211/abb2cefa-3c6f-4f5b-8814-9c38cd4d812d)

## Why?
This icon was hard-coded SVG. However, an icon representing the same meaning already exists in the `icons` package, so I thought it made sense to use it.

## Testing Instructions

I am a Windows user and tried to debug using Android Studio, but I just could not get the emulator to work properly.

The following icon should appear instead:

![new-ungroup](https://github.com/WordPress/gutenberg/assets/54422211/a4cee97d-8f00-4589-a80e-79ba3322e359)

I apologize for not being able to test this, but I would appreciate it if you could check to see if this change makes sense.
